Gemfile

```diff
@@ -57,8 +57,13 @@ group :development do
 end
 
 group :test do
+  gem 'rails-controller-testing'
   gem "webdrivers", "~> 4.4"
 end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem "tzinfo-data", platforms: %i[mingw mswin x64_mingw jruby]
+
+gem 'wdm', '>= 0.1.0' if Gem.win_platform?
```

Comment:
Interesting, why do we need it? Is Windows an inferior platform? :D

Reply:
Because it told me to basically: 
Please add the following to your Gemfile to avoid polling for changes:
    gem 'wdm', '>= 0.1.0' if Gem.win_platform?

It was red and ugly and this made it go away 😛